mflowgen/Tile_PE/construct.py

```diff
@@ -224,6 +224,7 @@ def construct():
   if pwr_aware:
       dc.update_params( { 'PWR_AWARE': parameters['PWR_AWARE'] }, True )
       init.update_params( { 'PWR_AWARE': parameters['PWR_AWARE'] }, True )
+      init.update_params( { 'flatten_effort': parameters['flatten_effort'] }, True )
       power.update_params( { 'PWR_AWARE': parameters['PWR_AWARE'] }, True )
```

Comment:
Should be executed unconditionally, same as other comment

Reply:
I have removed the condition on the PWR_AWARE variable. I have kept the condition on flatten_effort since that's only used by the scripts in power-domains.

Reply:
@ankita0805 did you forget to push? Or push to the wrong branches? The changes aren't here.

Reply:
https://github.com/StanfordAHA/garnet/blob/power-domains-tiles-flattened/mflowgen/Tile_PE/construct.py : L224-226?

Reply:
I see that but it's not showing in the PR for some reason...

Reply:
Looks like it may be a Github bug. I see that the changes are in the branch. Let's go ahead and merge.